app.rb

```diff
@@ -3,103 +3,113 @@
+  end
 
-  puts 'Select a person from the following list by number (Not ID): '
-  people.each_with_index do |person, index|
-    puts "#{index}) [#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+  def create_book()
+    print 'Title: '
+    title = gets.chomp
+    print 'Author: '
+    author = gets.chomp
+    books.push(Book.new(title, author))
+    puts 'Book created successfully.'
   end
-  person_input = gets.chomp.to_i
 
-  print 'Date: '
-  date = gets.chomp
+  def create_rental()
+    puts 'Select a book from the following list by number'
+    list_all_books
+    book_input = gets.chomp.to_i
 
-  rentals.push(Rental.new(date, people[person_input], books[book_input]))
-  puts 'Rental created successfully.'
-end
+    puts 'Select a person from the following list by number (Not ID): '
+    list_all_people
+    person_input = gets.chomp.to_i
+
+    print 'Date: '
+    date = gets.chomp
 
-def list_all_rentals_by_id(rentals)
-  print 'ID of person: '
-  person_id = gets.chomp.to_i
-  rentals.each do |rental|
-    puts "Date: #{rental.date}, Book: #{rental.book.title} by #{rental.book.author}" if rental.person.id == person_id
+    rentals.push(Rental.new(date, people[person_input], books[book_input]))
+    puts 'Rental created successfully.'
   end
-end
 
-def display_list
-  puts 'Welcome to School library App!'
-  puts "
-    Please choose an option by entering a number:
-    1- List all books
-    2- List all people
-    3- Create a person
-    4- Create a book
-    5- Create a rental
-    6- List all rentals for a given person id
-    7- Exit"
-end
+  def list_all_rentals_by_id()
+    print 'ID of person: '
+    person_id = gets.chomp.to_i
+    rentals.each do |rental|
+      puts "Date: #{rental.date}, Book: #{rental.book.title} by #{rental.book.author}" if rental.person.id == person_id
+    end
+  end
 
-def options(books, people, rentals)
-  loop do
-    display_list
-    option = gets.chomp.to_i
-    case option
-    when 1
-      list_all_books(books)
-    when 2
-      list_all_people(people)
-    when 3
-      create_person(people)
-    when 4
-      create_book(books)
-    when 5
-      create_rental(books, people, rentals)
-    when 6
-      list_all_rentals_by_id(rentals)
-    else
-      break
+  def display_list
+    puts 'Welcome to School library App!'
+    puts "
+      Please choose an option by entering a number:
+      1- List all books
+      2- List all people
+      3- Create a person
+      4- Create a book
+      5- Create a rental
+      6- List all rentals for a given person id
+      7- Exit"
+  end
+
+  def options()
+    loop do
+      display_list
+      option = gets.chomp.to_i
+      case option
+      when 1
+        list_all_books
+      when 2
+        list_all_people
+      when 3
+        create_person
+      when 4
+        create_book
+      when 5
+        create_rental
+      when 6
+        list_all_rentals_by_id
+      else
+        break
+      end
     end
```

Comment:
- [ ] It could have been preferable to place these codes in the main.rb file whose role is only to start the app and asked users for their initial choice of options(or to quit).